config.yml.erb

```diff
@@ -595,3 +595,5 @@ ec2_instance_id_endpoint: 'http://169.254.169.254/latest/meta-data/instance-id'
 # Credentials for OpenAI API
 openai_chat_completion_api_key: !Secret
 openai_chat_completion_org_id: !Secret
+openai_teaching_assistant_api_key:
+openai_teaching_assistant_org_id:
```

Comment:
If/when we develop any other "Teaching Assistant" features, would we want to have different API keys for them? If so, it might be worth being even more specific with these names. If not, LGTM!